crates/node/builder/Cargo.toml

```diff
@@ -62,6 +61,7 @@ reth-transaction-pool.workspace = true
 ## ethereum
 alloy-network.workspace = true
 alloy-primitives.workspace = true
+alloy-rpc-types = { workspace = true, features = ["engine"] }
```

Comment:
I could also use `alloy-rpc-types-eth`, `alloy-rpc-types-engine` and `alloy-serde` instead. Don't know how much granular you want to get.

Reply:
this is fine